packages/common/src/repository/dictionaryRepository.ts

```diff
@@ -0,0 +1,40 @@
+import { SelectedFields } from 'drizzle-orm/pg-core/query-builders/select.types';
+import { SQL } from 'drizzle-orm/sql';
+import { isEmpty } from 'lodash-es';
+
+import { GlobalConfig } from '../config/config.js';
+import { Dictionary, NewDictionary, dictionaries } from '../models/dictionaries.js';
+
+export default class DictionaryRepository extends GlobalConfig {
```

Comment:
same as the previous comment on classes vs functional composition

Reply:
changed code to functions